m4_sessions/1804inning/treating_depression_with_mobile_app.md

```diff
@@ -5,12 +5,39 @@ Andrew Jeffery
 Kailin Cannon
 Matt Renn
 
-Name of Session: 
+Name of Session: Treating Depression with Mobilie Apps
 
 Session Description (Why should students come to this session):
+This session will explore some the mobile treatment options as well as technological sources.  Students will leave with tools to help themselves an others.
 
 Session Outcomes (What will students learn/leave with?):
 
+* What can I do to help myself
+* I am now empowered to help others
+
 Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
+Pre Gear-Up:
+Send Resources
+
+* Intro:
+- Name and Mod (5 mins)
+- small group intros (5 mins)
+- Why did you choose this session?
+
+* Scope of the coversation (5 mins)
+- types of depression and what we are focusing on - Moderate, not severe.
+
+* Does technology lead to depression? (15 mins) - ANDREW
+
+* For moderate depression, when there's nothing else, is an app a good option?  Is this the second best option? (10 mins) - MATT
+
+* Mobile Treatment options (15 mins) - KAILIN, MEGAN
+- available apps: pros and cons 
+
+* what can we as students do - small group app brainstorming. (10 mins) - KAILIN, MEGAN
```

Comment:
Cool.  What could resources do you have a Turing?  What could you create for your personal project?  How can you support each other or the people in your families?